Doc/library/typing.rst

```diff
@@ -3612,14 +3688,10 @@ Aliases to asynchronous ABCs in :mod:`collections.abc`
 
    Deprecated alias to :class:`collections.abc.Coroutine`.
 
-   The variance and order of type variables
-   correspond to those of :class:`Generator`, for example::
-
-      from collections.abc import Coroutine
-      c: Coroutine[list[str], str, int]  # Some coroutine defined elsewhere
-      x = c.send('hi')                   # Inferred type of 'x' is list[str]
-      async def bar() -> None:
-          y = await c                    # Inferred type of 'y' is int
+   See
+   :ref:`annotating generators and coroutines <annotating-generators-and-coroutines>`
+   for details of using :class:`collections.abc.Coroutine` and this class
+   in type annotations.
```

Comment:
`typing.Coroutine` isn't actually a class; it's an alias. I think I'd go fo this:

```suggestion
   for details on using :class:`collections.abc.Coroutine` and
   ``typing.Coroutine`` in type annotations.
```